src/main/java/ch/njol/skript/expressions/ExprBarterOutput.java

```diff
@@ -0,0 +1,167 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.log.ErrorQuality;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.PiglinBarterEvent;
+import org.bukkit.inventory.ItemStack;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Barter Output")
+@Description("The items dropped by the piglin in a piglin bartering event.")
+@Examples({"on piglin barter:",
+			"\tif the bartering output contains a jack-o-lantern:",
+			"\t\tremove jack-o-lantern from bartering output",
+			"\t\tbroadcast \"Get out your amulets!\""})
+@Since("INSERT VERSION")
+public class ExprBarterOutput extends SimpleExpression<ItemType> {
+	
+	static {
+		Skript.registerExpression(ExprBarterOutput.class, ItemType.class,
+			ExpressionType.SIMPLE, "[the] [piglin] barter[ing] out(come|put)");
```

Comment:
```suggestion
			ExpressionType.SIMPLE, "[the] [piglin] barter[ing] output");
```
I would expect an outcome to be more of a success/failure state than the items you'll receive. I'd also like to see the syntax include the fact that these are items somehow.

Reply:
do you reckon using `drops` is a good idea?

Reply:
Sure, that could work.